### ISSUE TITLE
Problems when making a new migration

### ISSUE THREAD

**Issue by miguelramirezv** [prisma/migrate#628]
Hi Prisma Team! Prisma Migrate just crashed. 

## Versions
      
| Name        | Version            |
|-------------|--------------------|
| Platform    | darwin             | 
| Node        | v12.18.3           | 
| Prisma CLI  | 2.10.0             | 
| Binary      | af1ae11a423edfb5d24092a85be11fa77c5e499c| 

## Error
```
Error: Error in migration engine.
Reason: [libs/sql-schema-describer/src/walkers.rs:214:27] index out of bounds: the len is 0 but the index is 0

Please create an issue in the migrate repo with
your `schema.prisma` and the prisma command you tried to use 🙏:
https://github.com/prisma/migrate/issues/new

```

**Comment by tomhoule** [comment]
We worked on this problem and just released a fix in 2.10.1. Can you confirm that this does not happen anymore with the new version?

**Comment by tomhoule** [comment]
Other users confirmed the problem is gone for them in 2.10.1, so I'll close this as duplicate.
